packages/keymaps/src/browser/keymaps-frontend-contribution.ts

```diff
@@ -36,19 +39,20 @@ export namespace KeymapsCommands {
     export const OPEN_KEYMAPS_JSON: Command = {
         id: 'keymaps:openJson',
         category: 'Settings',
-        label: 'Open Keyboard Shortcuts (JSON)'
+        label: 'Open Keyboard Shortcuts (JSON)',
+        iconClass: 'theia-open-file-icon'
```

Comment:
I'm not sure this is the appropriate icon for the toolbar.
I'd expect this one even if it requires a CQ, it will be useful for other areas in Theia.

<div align='center'>

<img width="60" alt="Screen Shot 2019-06-06 at 11 33 36 AM" src="https://user-images.githubusercontent.com/40359487/59046165-8106a100-884f-11e9-89a7-529eec147d9c.png">


</div>

Reply:
Yep, a distinguishable icon would be nice